src/writeHelper.js

```diff
@@ -17,12 +17,16 @@ function writeLocal(agency, currentDateTime, data) {
   const version = "v1";
   const fileName = `/tmp/${agency}_${version}_${currentTimestamp}.json`
 
-  fs.writeFile(fileName, JSON.stringify(data), err => {
-    if (err) {
-      console.error(err);
-    }
-    console.log("wrote", fileName)
-  }); 
+  return new Promise((resolve, reject) => { 
+    fs.writeFile(fileName, JSON.stringify(data), err => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve();
+      }
+      console.log("wrote", fileName)
+    }); 
+  })
```

Comment:
💡 Suggestion:

Nodejs has [fs promises](https://nodejs.org/api/fs.html#promises-api) built in. If you `import { writeFile } from 'fs/promises'`, then

```suggestion
  return writeFile(fileName, JSON.stringify(data))
    .then((data) => {
      console.log("wrote", fileName);
      return data;
    });
```

This also changes the logic to only print `wrote example.json` when the file successfully wrote instead of every time (assuming that's what you want).

Reply:
Great suggestion, I'll commit that into the branch. Thank you!

I assumed that `reject(err)` would act similar to a return / throw and would end execution (and thus the `wrote example.json` wouldn't be hit) but even if that were true, this looks cleaner. 👍